src/main/java/edu/cornell/kfs/module/cg/document/validation/impl/AwardExtensionRule.java

```diff
@@ -223,7 +227,39 @@ protected boolean checkAccountNotUsedOnOtherAwards(AwardAccount awardAccount) {
 
 		return success;
 	}
-	
+
+    /**
+     * Checks for null on the nullable Billing Frequency Code field. Fixes NullPointerException.
+     */
+    @Override
+    protected boolean checkBillingFrequency() {
+        boolean success = true;
+
+        String newBillingFrequencyCode = newAwardCopy.getBillingFrequencyCode();
+        String oldBillingFrequencyCode = oldAwardCopy.getBillingFrequencyCode();
+
+        if (!org.apache.commons.lang3.StringUtils.equals(newBillingFrequencyCode, oldBillingFrequencyCode)) {

```

Comment:
I think it may be better to change the StringUtils import to org.apache.commons.lang3.StringUtils and make any needed changes.